code-scanning/properties/anchore.properties.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "Anchore Container Scan",
-    "creator": "Indeni Cloudrail",
```

Comment:
Why is the `creator` field removed?

Reply:
It is a bit unclear where this workflow originated. At one point, Anchore provided some workflows to GitHub via email when Actions were in the early stages. This workflow, however, was merged as part of a number of `code-scanning` workflows with [this commit](https://github.com/actions/starter-workflows/commit/52edf1b58088d3acb5c1444b65bf9988e1d1b498). Regardless, this workflow is specific to the [Anchore scan-action](https://github.com/anchore/scan-action) and has nothing to do as far as I can tell with the [Indeni Cloudrail product](https://cloudrail.app/). Removing this creator seems reasonable in order to more closely align the starter workflow with the organization involved in creating and maintaining the Action.

Reply:
Understandable. Then you can change the PR to have:
```
creator: "Anchore", 
```
And kindly remove the organization field

Reply:
@Phantsure thanks for getting back about this -- I'm a little confused, however. We recently contributed the [anchore-syft](https://github.com/actions/starter-workflows/blob/main/code-scanning/properties/anchore-syft.properties.json) starter workflow which has an `organization` field, and the commit template has this: 
```
Should include a matching code-scanning/properties/*.properties.json file (for example, [code-scanning/properties/codeql.properties.json](https://github.com/actions/starter-workflows/blob/main/code-scanning/properties/codeql.properties.json)), with properties set as follows:
 name: Name of the Code Scanning integration.
 organization: Name of the organization producing the Code Scanning integration.
 description: Short description of the Code Scanning integration.
 categories: Array of languages supported by the Code Scanning integration.
 iconName: Name of the SVG logo representing the Code Scanning integration. This SVG logo must be present in [the icons directory](https://github.com/actions/starter-workflows/tree/main/icons).
```
... which does not specify `creator` as a valid property. I'm happy to rename the `organization` to `creator`, but it seems like it's the wrong thing to do -- what do you think?

Reply:
<img width="325" alt="Screenshot 2022-09-14 at 11 56 09 AM" src="https://user-images.githubusercontent.com/25999504/190076149-96849b99-07a9-411b-9a7e-ff2768298bdf.png">
That must be missed by me. Sorry for that. But organization is not recognised as a field. Let me update the guide to have `creator` instead of `organization`. Thanks for informing

Reply:
Have raised the PR for above mentioned change. Can you also add organization to creator change for [anchore-syft](https://github.com/actions/starter-workflows/blob/main/code-scanning/properties/anchore-syft.properties.json) in this PR itself?

Reply:
Yes, thank you for the information! I've updated both files to use `creator` now.